### PR TITLE
Change default login background to cloud image

### LINKED ISSUE #33968: Login page default background needs to switch from the "icons" background to the "clouds" background
In theming, "Login image" needs to be changed to "Background image" and taken as default for the login page as well as people’s background image – until they change it.

### PR DESCRIPTION
**Pull request by CarlSchwan**
image

Fix #33968

### CODE REVIEW

**Review by szaimen — APPROVED** [review]
Looks great! :)

**Review by AndyScherzinger — APPROVED** [review]

File: apps/theming/css/settings-admin.scss [hunk 1]
```diff
@@ -101,7 +101,7 @@
         margin-bottom: 20px;
         cursor: pointer;
         background-color: var(--color-primary);
-        background-image: var(--image-background, var(--image-background-plain, url('../../../core/img/background.svg'), linear-gradient(40deg, #0082c9 0%, #30b6ff 100%)));
+        background-image: var(--image-background, var(--image-background-plain, url('../../../core/img/app-background.jpg'), linear-gradient(40deg, #0082c9 0%, #30b6ff 100%)));
```
Comment (jancborchardt) [inline apps/theming/css/settings-admin.scss:104]: Looks great, and additionally the fallback could be changed from the  linear-gradient to the plain color `#0082c9` (our primary)? @CarlSchwan

File: core/css/guest.css [hunk 2]
```diff
@@ -21,14 +21,11 @@ body {
...
-	color: var(--color-primary-text);
+	color: var(--color-text);
 	text-align: center;
 	background-color: var(--color-primary);
-	background-image: var(--image-background, var(--image-background-plain, url('../../../core/img/background.svg'), linear-gradient(40deg, #0082c9 0%, #30b6ff 100%)));
-	background-position: 50% 50%;
-	background-repeat: repeat;
-	background-size: var(--image-background-size, 275px, contain);
-	background-attachment: fixed; /* fix background gradient */
+	background-image: var(--image-background, var(--image-background-plain, url('../../../core/img/app-background.jpg'), linear-gradient(40deg, #0082c9 0%, #30b6ff 100%)));
```
Comment (jancborchardt) [inline core/css/guest.css:27]: Same here

File: core/css/guest.css [hunk 3]
```diff
@@ -21,14 +21,11 @@ body {
 	font-size: .875em;
 	line-height: 1.6em;
 	font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen-Sans, Ubuntu, Cantarell, 'Helvetica Neue', Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol';
-	color: var(--color-primary-text);
+	color: var(--color-text);
 	text-align: center;
 	background-color: var(--color-primary);
-	background-image: var(--image-background, var(--image-background-plain, url('../../../core/img/background.svg'), linear-gradient(40deg, #0082c9 0%, #30b6ff 100%)));
-	background-position: 50% 50%;
```
Comment (jancborchardt) [inline core/css/guest.css:28]: The `background-position: center center;` should stay though so it’s exactly like when logged in?

**Pull request merged**